pygromos/simulations/hpc_queuing/job_scheduling/schedulers/simulation_scheduler.py

```diff
@@ -21,7 +21,7 @@ def do(in_simSystem: Gromos_System,
        simulation_run_num: int,
        equilibration_run_num: int = 0,
        work_dir: str = None,
-       initialize_first_run= False, reinitialize_every_run= False,
+       initialize_first_run= True, reinitialize_every_run= False,
```

Comment:
I'm against this change. I believe PyGromos should by default not change anything.